client/graphql-apollo/index.js

```diff
@@ -0,0 +1,23 @@
+const { ApolloServer, gql } = require('apollo-server');
+
+// Define the GraphQL schema
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`;
+
+// Define the resolvers
+const resolvers = {
+  Query: {
+    hello: () => 'Hello World!',
+  },
+};
+
+// Create a new ApolloServer instance
+const server = new ApolloServer({ typeDefs, resolvers });
+
+// Start the server
+server.listen().then(({ url }) => {
+  console.log(`Server ready at ${url}`);
+});
```

Comment:
This may be a work in progress. The actual code has to go into _src/route/auth_ directory

Reply:
This was a simple testing server, it has been deleted.